independent-publisher-connectors/CO2 Signal/apiProperties.json

```diff
@@ -0,0 +1,82 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",

```

Comment:
@paulculmsee You have agreed to not have this iconBrandColor for your connector. And there is a default iconBrandcolor for all the independent publisher connectors, i.e. `"#da3b01"`, Please refer the [templates](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json). 

![image](https://user-images.githubusercontent.com/56276394/147993531-ad88a11d-cbf0-4ac6-a63f-47559a5ec35b.png)

Reply:
Sorry misread it and assumed 007ee5 was valid